NSS-Day-Cohort-49/giffygram-dramatic-chipmunks
# added necessary missing modules

#### Changes Made
1. Added PostEntry.js / Post.js / MessageList.js / NavBar.js modules. 
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout pr-adding-necessary-modules
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user does BLANK, then BLANK should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
